jobclient/pom.xml

```diff
@@ -3,7 +3,7 @@
   <groupId>twosigma</groupId>
   <artifactId>cook-jobclient</artifactId>
   <packaging>jar</packaging>
-  <version>0.2.0-SNAPSHOT</version>
+  <version>0.1.2-SNAPSHOT</version>
```

Comment:
Was the previous version from this `pom.xml` never used?

Reply:
Apparently. `0.1.2-SNAPSHOT` is the version number that was in project.clj.